explorer/src/api/mod.rs

```diff
@@ -8,6 +8,9 @@ use jormungandr_lib::interfaces::{Cors, Tls};
 use std::{net::SocketAddr, time::Duration};
 use warp::{http::Response as HttpResponse, Filter, Rejection, Reply};
 
+const QUERY_DEPTH_LIMIT: usize = 15;
```

Comment:
Is it possible to put it somewhere in config struct or something similar, if it is presented of course 😊?